packages/accordion/src/AccordionContainer.spec.tsx

```diff
@@ -31,22 +36,40 @@ describe('AccordionContainer', () => {
     expect(triggerButton).not.toHaveAttribute('disabled');
   });
 
-  const sections = Array(3)
-    .fill(undefined)
-    .map((s, i) => i);
-  const CONTAINER_ID_PREFIX = 'test';
-
-  const BasicExample = (props: IUseAccordionProps = {}) => (
-    <AccordionContainer idPrefix={CONTAINER_ID_PREFIX} {...props}>
+  const BasicExample = (props: Omit<IUseAccordionProps<number>, 'sections'> = {}) => (
+    <AccordionContainer idPrefix={CONTAINER_ID_PREFIX} sections={sections} {...props}>
       {({ getHeaderProps, getTriggerProps, getPanelProps }) => (
         <>
-          {sections.map((section, index) => {
+          {sections.map(section => {
             return (
               <div key={section}>
-                <div {...getHeaderProps({ ariaLevel: 1, 'data-test-id': 'header' })}>
-                  <div {...getTriggerProps({ index, 'data-test-id': 'trigger' })}>Trigger</div>
+                <div
+                  {...getHeaderProps({
+                    'aria-level': 1,
+                    // @ts-expect-error data-test-id for testing
```

Comment:
Please remove all instances of `@ts-expect-error` in this file by restructuring like `<div data-test-id="header" {...getHeaderProps({ ... })}>`